spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -859,6 +906,7 @@ class CometExecSuite extends CometTestBase {
         .saveAsTable("bucketed_table2")
 
       withSQLConf(
+        "spark.comet.exec.sort_merge_join.disabled" -> "true",
```

Comment:
This test explicitly checks for Spark sort merge join. I'd like to keep what it proposes to test so disable Comet sort merge join here.